mne/baseline.py

```diff
@@ -41,15 +41,18 @@ def rescale(data, times, baseline, mode='mean', copy=True, verbose=None):
         and if ``bmax is None`` then ``bmax`` is set to the end of the
         interval. If baseline is ``(None, None)`` the entire time
         interval is used. If baseline is None, no correction is applied.
-    mode : None | 'ratio' | 'zscore' | 'mean' | 'percent' | 'logratio' | 'zlogratio'
-        Do baseline correction with ratio (power is divided by mean
-        power during baseline) or zscore (power is divided by standard
-        deviation of power during baseline after subtracting the mean,
-        power = [power - mean(power_baseline)] / std(power_baseline)), mean
-        simply subtracts the mean power, percent is the same as applying ratio
-        then mean, logratio is the same as mean but then rendered in log-scale,
-        zlogratio is the same as zscore but data is rendered in log-scale
-        first.
+    mode : 'mean' | 'ratio' | 'logratio' | 'percent' | 'zscore' | 'zlogratio' |
+           None
```

Comment:
This (docstrings with long arg lists) to me is a good time to make a PEP8 exception -- so you can keep on one line and just add `# noqa: E501` at the end of the `"""` to make flake checkers live with it

Reply:
I'm not sure I can live with it 😄 , so I tried to fit everything into 79 characters. However, if there is no easy way around it I'll add the flag back.

Reply:
Yes it renders okay with Sphinx but the raw code view is not as good, which people need to use also (this is also a break from how we do it everywhere else so it's a consistency thing, too -- if we want to change it we should probably discuss it elsewhere since it's not the focus of this PR anyway)